traffic_portal/app/src/common/modules/form/serverCapability/view/FormViewServerCapabilityController.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+var FormViewServerCapabilityController = function(serverCapability, $scope, $controller, $uibModal, $anchorScroll, locationUtils, serverCapabilityService) {
+
+	// extends the FormServerCapabilityController to inherit common methods
+	angular.extend(this, $controller('FormServerCapabilityController', { serverCapability: serverCapability, $scope: $scope }));
+
+	var deleteServerCapability = function(serverCapability) {
+		serverCapabilityService.deleteServerCapability(serverCapability.name)
+			.then(function() {
+				locationUtils.navigateToPath('/server-capabilities');
+			});
+	};
+
+	$scope.serverCapabilityName = angular.copy(serverCapability.name);
```

Comment:
I don't think you need to use `angular.copy` here.

Reply:
yep i think you are right because you can't change a server cap name even if you wanted to